submissions/EmmanuelIdiaghe/html/html-forms/product-page.html

```diff
@@ -19,12 +19,12 @@ <h3>...your number 1 online shop!</h3><hr>
         <div>
             <p><strong>WIRELESS HEADSET:</strong></p>
             <figure>
-            <img src="headset.jpg" height="100px" alt="white-headset-picture">
+            <img src="C:\Users\Emancipator\Desktop\WEB DEV\Dufuna\Dufuna-CodeCamp20\submissions\EmmanuelIdiaghe\html\images/headset.jpg" height="100px" alt="white-headset-picture">
```

Comment:
This path isn't relative. If I decide to view your work on my computer, I won't be able to cos I don't have a file path like `C:\Users\Emancipator\Desktop\WEB DEV\Dufuna\`.  So that been said. how best can you handle this